src/watch.ts

```diff
@@ -64,6 +64,12 @@ export class Watch {
                 // ignore parse errors
             }
         });
+        stream.on('error', (err) => {
+            done(err);
+        });
+        stream.on('close', () => {
+            done(null);
```

Comment:
is this related to it closing on the serverside? if so I think we should be passing a custom error object to the `done()` function

Reply:
This would be a server-side close, but I think that's an expected case? I don't think we want to return an error there.